docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,407 @@
+    - Strict MIME type for module scripts: Loading module script from the local file feature has been removed.
+    - App control URL bug fix: appcontrol:// scheme now able to launch application from encoded URL.
+- Web runtime
+  - Device API has been added for wrt-service.
+  - Web server and web server has been added as a wrt-service built-in module.
+  - WRT.js API management policy has been added.
+    - API has been changed from generic js to type script.
+    - API compatibility checker has been added.
+
+#### Fixes
+
+- Compatibility enhancement
+  - The problem of autofill operation in major web site has been improved.
+- Stability enhancement
+  - Unexpected random crash while clean up sequence has been fixed.
+
+
+### Lightweight Web Solution
+
+#### New and changed features
+
+- Rendering engine
+  - Data Element's DOM API has been added.
+  - Performance Web API set has been added.
+  - Range Web API set has been added.
+- JS Engine	
+  - ES10 features have been added.
+    - Array.prototype.flat
+    - Array.prototype.flatMap
+    - Object.fromEntries
+    - String.prototype.trimStart
+    - String.prototype.trimEnd
+  - Following features' syntax has been updated.
+    - Optional catch binding parameters
+    - Allowing U+2028 (LINE SEPARATOR) and U+2029 (PARAGRAPH SEPARATOR) in string literals
+  - Array.prototype.sort has been updated due to the specification change.
+  - Function.prototype.toString has been updated due to the specification change.
+- Performance and Memory Optimization
+  - JS Engine
+    - Debugger core features have been updated.
+    - Lazy AtomicString initialization for memory/performance has been optimized.
+    - Vector extension logic has been improved.
+
+#### Fixes
+
+- Bug fixes
+  - Layout bugs have been fixed.
+  - Scroll bugs have been fixed.
+  - Font Rendering bugs have been fixed.
+  - Memory overflow bugs have been fixed.
+  - Crash issues have been fixed.
+
+
+### Tizen .NET
+
+#### New and changed features
+
+- NET runtime and tools
+  - .NETCore has been upgraded from 3.1.0 to 3.1.3.
+- TizenFx
+  - Tizen.NUI
+    - VertexBuffer class has been added.
+    - Pagination class has been added.
+    - Screen connection methods have been added to CustomView class.
+    - SlidingStarted event has been added to Slider class.
+    - ThumbColor property has been added to Slider class.
+    - FlexLayout and GridLayout have been added.
+    - ScrollPosition and ScrollCurrentPosition have been added to ScrollableBase class.
+    - Notification class has been added.
+    - FlexContainer class has been deprecated.
+    - IndicatorColor and SelectedIndicatorColor have been added to Pagination class.
+    - Quickpanel has been added to WindowSystem.
+  - Tizen.Network.Connection
+    - Required privilege level has been changed.
+  - Tizen.MachineLearning
+    - Custom filter class has been added.
+  - Tizen.Multimedia
+    - Setter/Getter for volume recording has been added to AudioIO.
```

Comment:
Can we write like: 
- Setter and Getter...

Reply:
Will accept